Students/Pop Calin/First assignments/FibonacciRecursive.java

```diff
@@ -0,0 +1,39 @@
+package assignments;
+
+public class FibonacciRecursive 
```

Comment:
This is not a recursive implementation.

>A recursive algorithm is an algorithm which calls itself with "smaller (or simpler)" input values, and which obtains the result for the current input by applying simple operations to the returned value for the smaller (or simpler) input.

What you have done is an iterative approach.